README.md

```diff
@@ -1 +1,22 @@
 # hw01
+#<Biography of David Kanos>
```

Comment:
You need as space between the formatting and your text, otherwise the formatting won't apply.